getting-started/create-catalog.sh

```diff
@@ -0,0 +1,37 @@
+#
+# Copyright (c) 2024 Snowflake Computing Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+PRINCIPAL_TOKEN="principal:root;realm:default-realm"
```

Comment:
whats the stance on using this credential? Since this is a demo, I think it's acceptable

Reply:
I can't tell whether this works by design or due to a bug...